js/greeter.js

```diff
@@ -228,6 +228,8 @@ class AntergosTheme {
 
 			if ( lightdm[ cmd ] ) {
 				$( template ).appendTo( $( this.$actions_container ) ).click( action, ( event ) => {
+					// prevent accidential hibernates after resume 
```

Comment:
initialize() will not be called after a resume from hibernate (at least not on my test systems ;)).
So I would suggest adding `$( this.$actions_container ).children().blur();` before calling the corresponding lightdm methods.